### ISSUE TITLE
Request is retransmitted after receiving reply

### ISSUE THREAD

**Issue by agranig** [kirm/sip.js#11]
When I send a REGISTER to another proxy using `send(msg, cb)`, I get back 100 and 401. For some reason, sip.js  sends another REGISTER with the same cseq as the first one (a retransmit) ~90-100ms after it sent the first request, although it received a 100 and even a 401 in the meanwhile. The same happens when I increase cseq and send the message again including Authorize header from within my callback, just that I get a 100 back there, then sip.js retransmits the authenticated request, then I get the 200 and it stops.

So to sum it up, each request seems to be retransmitted once, even though we already got a reply. Is there some way to manually stop a specific transaction, or is it just a timing issue?
